node/turbo/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "author": {
+    "name": "Parsa Ghadimi",
+    "email": "me@qti3e.com",
+    "url": "http://qti3e.com"
+  },
+  "dependencies": {
+    "physical-cpu-count": "^2.0.0",
```

Comment:
@qti3e I prefer to use `~` range instead of `^`. `^` allows **minor** upgrade which I prefer not to, in order to display **minor** (`X`.`Y`) on `README`

Reply:
@qti3e -`physical-cpu-count` is not even needed/used here, since there is no clustering going on and thus no need to count CPUs. If, at all, this should be done using the native option instead of a dependency.

@waghanza -By default, `npm i` will always use `^`, this is the preferred choice, most of the time, since it will guarantee that the latest code is always installed. As for showing the version in the readme, just show `major`.`minor` and avoid the `patch` version (third number).

Reply:
@waghanza done